Cannot find namespace 'google'

Hi,

first of all, this seems like a really great plugin. Nice work!

When I add the plugin to my Angular Ionic project, I get the error "Cannot find namespace 'google'".

```
[ng] Error: node_modules/capacitor-native-google-one-tap-signin/dist/esm/GoogleOneTapAuth.d.ts:14:110 - error TS2503: Cannot find namespace 'google'.
[ng] 
[ng] 14     renderSignInButton(parentElementId: string, options: RenderSignInButtonOptions, gsiButtonConfiguration?: google.GsiButtonConfiguration): Promise<SuccessSignInResult>;
[ng]                                                                                                                 ~~~~~~
[ng] 
[ng] 
[ng] Error: node_modules/capacitor-native-google-one-tap-signin/dist/esm/GoogleOneTapAuth.d.ts:15:112 - error TS2503: Cannot find namespace 'google'.
[ng] 
[ng] 15     _doRenderSignInButton(parentElementId: string, options: RenderSignInButtonOptions, gsiButtonConfiguration: google.GsiButtonConfiguration | undefined, androidAndIosButtonElem: HTMLElement): Promise<SuccessSignInResult>;
[ng]                                                                                                                   ~~~~~~
[ng] 
[ng] 
[ng] Error: node_modules/capacitor-native-google-one-tap-signin/dist/esm/definitions.d.ts:139:110 - error TS2503: Cannot find namespace 'google'.
[ng] 
[ng] 139     renderSignInButton(parentElementId: string, options: RenderSignInButtonOptions, gsiButtonConfiguration?: google.GsiButtonConfiguration): Promise<SuccessSignInResult>;
[ng]
```

What helps is adding the package `@types/google-one-tap` you are using in the plugin to my dev dependencies and adding `/// <reference types="@types/google-one-tap" />` to `node_modules/capacitor-native-google-one-tap-signin/dist/esm/definitions.d.ts`.

Regards
Nicolas

--- a/package.json
+++ b/package.json
@@ -71,7 +71,7 @@
     "rimraf": "^4.1.2",
     "rollup": "^2.32.0",
     "swiftlint": "^1.0.1",
-    "typescript": "^4.9.5"
+    "typescript": "^5.1.6"
   },
   "peerDependencies": {
     "@capacitor/core": "^5.0.0"

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,4 +1,6 @@
 import { GoogleOneTapAuth } from './GoogleOneTapAuth';
 
+export type * as google from 'google-one-tap';
+
 export * from './definitions';
 export { GoogleOneTapAuth };

